conf/messages.en

```diff
@@ -523,7 +523,7 @@ supplementary.addDocument.issuingAuthorityName = Issuing authority name
 supplementary.addDocument.issuingAuthorityName.error.length = Issuing authority name is too long
 supplementary.addDocument.dateOfValidity = Date of validity
 supplementary.addDocument.measurementUnit = Measurement unit and qualifier
-supplementary.addDocument.measurementUnit.error.length = Measurement unit and qualifier has to be either 3 or 4 characters long
+supplementary.addDocument.measurementUnit.error.length = Measurement unit and qualifier has to be either 3 to 5 characters long
```

Comment:
Doesn't really scan as a sentence.  Has this been recommended by design?

Reply:
yes, I missed this from the ticket :)